hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/metrics/OzoneManagerStateMachineMetrics.java

```diff
@@ -63,16 +62,6 @@ public static synchronized OzoneManagerStateMachineMetrics create() {
     }
   }
 
-  @VisibleForTesting
-  public long getApplyTransactionMapSize() {
-    return applyTransactionMapSize.value();
```

Comment:
applyTransactionMapSize and ratisTransactionMapSize is not more required to be exposed now, we can expose number of double buffer transaction as pending transactions `ozoneManagerDoubleBuffer.getFlushedTransactionCount()`

Reply:
Let's do the metrics improvement separately.   This change is already quite big and complicated.

Reply:
Filed HDDS-10126 for cleaning up the metrics.